src/repos/UserRepo.ts

```diff
@@ -84,7 +84,17 @@ const getUserByNetID = async (netID: string): Promise<User | undefined> => {
 };
 
 const getUsers = async (): Promise<User[]> => {
-  return db().find();
+  return db().find({
+    relations: [
+      'matches',
+      'matches.users',
+      'matches.schedule',
+      'matches.schedule.times',
+      'clubs',
+      'interests',
+      'major',
+    ],
```

Comment:
Btw @chalo2000, I added this to your `getUsers` function because we want to load users with their foreign key relationships. If `relations` isn't specified, the serialized user will show undefined for any foreign key (think: club, interest, major, etc).